src/params/index.ts

```diff
@@ -134,6 +134,7 @@ export function defineFloat(name: string, options: ParamOptions<number> = {}): F
 /**
  * Declare a list param (array of strings).
  *
+ * @hiddden
```

Comment:
I don't think hidden works 🤔. You need to make it @internal or just get rid of the the implementation.